core/modules/date/views/date.views.inc

```diff
@@ -60,7 +54,7 @@ function date_views_views_plugins() {
 /**
  * Implements hook_views_data().
  */
-function date_views_views_data() {
+function date_views_data() {
```

Comment:
Yikes, this looks like a big fix! This might be worth separating into a stand-alone PR.